src/Components/Lobby.tsx

```diff
@@ -109,6 +106,10 @@ const Lobby = () => {
     }
   }
 
+  useEffect(() => {
```

Comment:
normally other devs would comment here and ask to remove unused code.
But we can cleanup later :D since ive done the same thing hehe